cla_public/templates/online-safety.html

```diff
@@ -3,43 +3,61 @@
-  <p class="govuk-body">{{ _('You can improve your safety by deleting your history. But you need to be careful because:') }}</p>
+  <h2 class="govuk-heading-l">{{ _('How to delete cookies and your browsing history') }}</h2>
+  <p class="govuk-body">{{ _('Be careful because:') }}</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>{{ _('deleting cookies will also get rid of stored passwords for online accounts') }}</li>
+    <li>{{ _('deleting cookies will also delete stored passwords for online accounts') }}</li>
     <li>{{ _('clearing your history could make someone more suspicious') }}</li>
   </ul>
-  <p class="govuk-body">{{ _('So just remove information about the websites you want to keep private. You can do that on:') }}</p>
+  <p class="govuk-body">{{ _('Try to only remove information about the websites you want to keep private.') }}</p>
+  <p class="govuk-body">{{ _('Find out how to do that on:') }}</p>
+
   <ul class="govuk-list govuk-list--bullet">
     <li><a class="govuk-link" href="https://support.google.com/chrome/answer/95589?hl=en&rd=1" rel="external">{{ _('Google Chrome') }}</a></li>
     <li><a class="govuk-link" href="http://windows.microsoft.com/en-gb/windows-vista/clear-the-history-of-websites-youve-visited" rel="external">{{ _('Internet Explorer') }}</a></li>
     <li><a class="govuk-link" href="https://support.mozilla.org/en-US/kb/delete-browsing-search-download-history-firefox" rel="external">{{ _('Mozilla Firefox') }}</a></li>
     <li><a class="govuk-link" href="https://support.apple.com/kb/PH19215?locale=en_US&viewlocale=en_US" rel="external">{{ _('Safari') }}</a></li>
-    <li><a class="govuk-link" href="http://www.opera.com/help/tutorials/security/sharing/" rel="external">{{ _('Opera') }}</a></li>
+    <li><a class="govuk-link" href="https://www.samsung.com/uk/support/mobile-devices/using-the-samsung-internet-app/" rel="external">{{ _('Samsung internet') }}</a></li>
+    <li><a class="govuk-link" href="https://support.apple.com/en-gb/HT201265#:~:text=To%20clear%20your%20history%20and,t%20change%20your%20AutoFill%20information." rel="external">{{ _('Iphone') }}</a></li>
   </ul>
 
-  <h2 class="govuk-heading-l">{{ _('Browsing in private') }}</h2>
+  <h2 class="govuk-heading-l">{{ _('How the ‘Exit this page’ button works on this website') }}</h2>
   <p class="govuk-body">
-    {{ _('You can also look at a website without information like cookies being stored by using the private browsing setting
-    in your browser. Go to the menu, click on ‘File’ and choose:') }}
+    {{ _('There is an ‘Exit this page’ button on some pages. If you click this button, all the information you’ve entered will be deleted. This website will close.')}}
   </p>
+  <p class="govuk-body">
+    {{ _('You’ll automatically go to the Google search homepage:') }} <a class="govuk-link" href="https://www.bbc.co.uk/weather" rel="external">{{ _('https://www.bbc.co.uk/weather') }}</a>
```

Comment:
You're still saying they're going to Google search homepage, when it's the BBC Weather.

`You'll automatically go to the BBC Weather homepage:`

Reply:
Thanks Pat, missed that one! :)